<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Agregar</class>
 <widget class="QDialog" name="Agregar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QStackedWidget" name="area">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="page">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Transaccion_ID</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="transaction_id">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Titulo :*</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="titulo"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Clientes :* </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="cliente"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Tipo *</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="type">
                 <item>
                  <property name="text">
                   <string/>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>sell</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>lend</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>return</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Finalizado</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="finished">
                 <property name="text">
                  <string>FALSE</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_3">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Cientes</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Nombre:* </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="name">
                 <property name="maxLength">
                  <number>50</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Email:* </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="email">
                 <property name="maxLength">
                  <number>100</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Cumplea√±os:  </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateEdit" name="birthdate">
                 <property name="displayFormat">
                  <string>yyyy-MM-dd</string>
                 </property>
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>Genero: </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="gender">
                 <item>
                  <property name="text">
                   <string>F</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>M</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>Activo: </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="active">
                 <property name="text">
                  <string>True</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_4">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_14">
               <item>
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>Libro</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="book_id_2">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_15">
               <item>
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>Autor:* </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="author_id"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_16">
               <item>
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>Titulo:*</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="title">
                 <property name="maxLength">
                  <number>100</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_17">
               <item>
                <widget class="QLabel" name="label_18">
                 <property name="text">
                  <string>A√±o:* </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateEdit" name="year">
                 <property name="displayFormat">
                  <string>yyyy</string>
                 </property>
                 <property name="calendarPopup">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_18">
               <item>
                <widget class="QLabel" name="label_19">
                 <property name="text">
                  <string>Lenguaje:*</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="language">
                 <property name="maxLength">
                  <number>2</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_19">
               <item>
                <widget class="QLabel" name="label_20">
                 <property name="text">
                  <string>URL:  </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="cover_url">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="maxLength">
                  <number>500</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_20">
               <item>
                <widget class="QLabel" name="label_21">
                 <property name="text">
                  <string>Precio:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="price">
                 <property name="decimals">
                  <number>2</number>
                 </property>
                 <property name="maximum">
                  <double>9999.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_13">
               <item>
                <widget class="QLabel" name="label_26">
                 <property name="text">
                  <string>Copias:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="copias">
                 <property name="decimals">
                  <number>0</number>
                 </property>
                 <property name="minimum">
                  <double>1.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>9999.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_21">
               <item>
                <widget class="QLabel" name="label_22">
                 <property name="text">
                  <string>Descripccion:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="desciption">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="maxLength">
                  <number>500</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_5">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_22">
               <item>
                <widget class="QLabel" name="label_23">
                 <property name="text">
                  <string>Autor</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="author_id_2">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_23">
               <item>
                <widget class="QLabel" name="label_24">
                 <property name="text">
                  <string>Nombre:* </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="name_2">
                 <property name="maxLength">
                  <number>100</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_24">
               <item>
                <widget class="QLabel" name="label_25">
                 <property name="text">
                  <string>Nacionalidad:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="nationality">
                 <item>
                  <property name="text">
                   <string>MEX</string>
                  </property>
                  <property name="icon">
                   <iconset resource="iconos.qrc">
                    <normaloff>:/new/prefix1/iconos/mexico.png</normaloff>:/new/prefix1/iconos/mexico.png</iconset>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>ARG</string>
                  </property>
                  <property name="icon">
                   <iconset resource="iconos.qrc">
                    <normaloff>:/new/prefix1/iconos/argentina.png</normaloff>:/new/prefix1/iconos/argentina.png</iconset>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>USA</string>
                  </property>
                  <property name="icon">
                   <iconset resource="iconos.qrc">
                    <normaloff>:/new/prefix1/iconos/estados-unidos-de-america.png</normaloff>:/new/prefix1/iconos/estados-unidos-de-america.png</iconset>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>FRA</string>
                  </property>
                  <property name="icon">
                   <iconset resource="iconos.qrc">
                    <normaloff>:/new/prefix1/iconos/francia.png</normaloff>:/new/prefix1/iconos/francia.png</iconset>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>BRA</string>
                  </property>
                  <property name="icon">
                   <iconset resource="iconos.qrc">
                    <normaloff>:/new/prefix1/iconos/brazil.png</normaloff>:/new/prefix1/iconos/brazil.png</iconset>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>CHN</string>
                  </property>
                  <property name="icon">
                   <iconset resource="iconos.qrc">
                    <normaloff>:/new/prefix1/iconos/china.png</normaloff>:/new/prefix1/iconos/china.png</iconset>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>DEU</string>
                  </property>
                  <property name="icon">
                   <iconset resource="iconos.qrc">
                    <normaloff>:/new/prefix1/iconos/alemania.png</normaloff>:/new/prefix1/iconos/alemania.png</iconset>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>COL</string>
                  </property>
                  <property name="icon">
                   <iconset resource="iconos.qrc">
                    <normaloff>:/new/prefix1/iconos/colombia.png</normaloff>:/new/prefix1/iconos/colombia.png</iconset>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>ESP</string>
                  </property>
                  <property name="icon">
                   <iconset resource="iconos.qrc">
                    <normaloff>:/new/prefix1/iconos/espana.png</normaloff>:/new/prefix1/iconos/espana.png</iconset>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>CAN</string>
                  </property>
                  <property name="icon">
                   <iconset resource="iconos.qrc">
                    <normaloff>:/new/prefix1/iconos/canada.png</normaloff>:/new/prefix1/iconos/canada.png</iconset>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string/>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="cancelar">
           <property name="text">
            <string>Cancelar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="agregar">
           <property name="text">
            <string>Agregar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>* = NOT NULL</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="iconos.qrc"/>
 </resources>
 <connections/>
</ui>
